neutone_sdk/non_realtime_wrapper.py

```diff
@@ -361,6 +361,14 @@ def set_progress_percentage(self, progress_percentage: float) -> None:
             ), "Progress percentage must be between 0 and 100"
         self.progress_percentage = progress_percentage
 
+    def has_progress_percentage(self) -> bool:
+        """
+        Returns True if the model sets the progress percentage of the model during
+        forward pass.
+        If this is False and the model is a oneshot model, the plugin should estimate the progress based on last run.
+        """
+        return True
```

Comment:
It might make more sense for this to be False by default, but I think True is fine for now.